src/App/Extension.h

```diff
@@ -24,11 +24,11 @@
 #ifndef APP_EXTENSION_H
 #define APP_EXTENSION_H
 
-#include "PropertyContainer.h"

```

Comment:
Removing PropertyContainer.h causes the build failure.
There is also a cyclic include of Extension.h and ExtensionContainer.h. It should be checked if it can be avoided to include Extension.h in ExtensionContainer.h and instead a forward declaration be used.